File: app/src/main/java/com/noalecohen/dispatcher/LogisterActivity.java
package com.noalecohen.dispatcher;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;

public class LogisterActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_logister);
        FragmentManager fragmentManager = getSupportFragmentManager();
        fragmentManager.beginTransaction().replace(R.id.startup_frame_content, new SplashFragment()).commit();
    }
}

File: app/src/main/java/com/noalecohen/dispatcher/LogisterActivity.kt
package com.noalecohen.dispatcher

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class LogisterActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_logister)
        val fragmentManager = supportFragmentManager
        fragmentManager.beginTransaction().replace(R.id.startup_frame_content, SplashFragment())
            .commit()
    }
}